app/src/test/kotlin/com/gravatar/app/di/AppModuleTest.kt

```diff
@@ -10,6 +11,8 @@ class AppModuleTest : KoinTest {
     @OptIn(KoinExperimentalAPI::class)
     @Test
     fun checkAllModules() {
-        appModule.verify()
+        appModule.verify(
+            extraTypes = listOf(Context::class)
```

Comment:
This test is failing because it has a missing `Context` type for the `FileUtils` constructor. The problem is only here in the test, so I assume that should be fine, but if you know about a better way to handle it, let me know!